go.mod

```diff
@@ -3,14 +3,13 @@ module github.com/lightninglabs/loop
 require (
 	github.com/btcsuite/btcd v0.0.0-20190315201642-aa6e0f35703c
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/btcutil v0.0.0-20190207003914-4c204d697803
+	github.com/btcsuite/btcutil v0.0.0-20190316010144-3ac1210f4b38
 	github.com/coreos/bbolt v1.3.2
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/golang/protobuf v1.3.1
 	github.com/grpc-ecosystem/grpc-gateway v1.8.5
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/neutrino v0.0.0-20190314214430-643615b8c132 // indirect
-	github.com/lightningnetwork/lnd v0.0.0-20190314214430-b4a1024ac74fc576c65c8074288a5ffaf6bd1ec4
+	github.com/lightningnetwork/lnd v0.5.1-beta.0.20190322220528-6ad8be25e1aa
```

Comment:
Should't this be 0.5.2? Not sure how `go mod` handles internal release/tag branching.

Reply:
This is what it adds if I specify the commit `6ad8be25e1aa`